src/beeswarm.jl

```diff
@@ -0,0 +1,50 @@
+# ---------------------------------------------------------------------------
+# Beeswarm plot
+@recipe function f(::Type{Val{:beeswarm}}, x, y, z; trim::Bool=false, side::Symbol=:both)
+    if !(side in [:both :left :right])
+        warn("side (you gave :$side) must be one of :both, :left, or :right")
+        side = :both
+        info("side set to :$side")
+    end
+    x, y = Float64[], Float64[]
+    glabels = sort(collect(unique(x)))
+    bw = d[:bar_width]
+    bw == nothing && (bw = 0.8)
+    for (i,glabel) in enumerate(glabels)
+
+        # We get the values for this label
+        lab_y = y[filter(i -> _cycle(x,i) == glabel, 1:length(y))]
+        lab_x = zeros(lab_y)
+        
+        # Then we apply Sturge's rule to get the number of bins
```

Comment:
Yes, I am still waiting for the PR author to submit the new page on histogram functionality to PlotDocs that he promised :-) so you'd have to actually type `plotattr("bins")` to notice this at the moment (or read the code).